intake/tests.py

```diff
@@ -1,3 +1,334 @@
+        request = request_factory.get('')
+        response = views.urgency(request, 12)
+        self.assertEqual(response.status_code, 200)
+
+    def test_urgency_post_request(self):
+        request_factory = RequestFactory()
+        post_data = {
+            'urgency': 'true'
+        }
+        request = request_factory.post('', post_data)
+        response = views.urgency(request, 12)
+        self.assertEqual(response.status_code, 302)
+
+        request_factory = RequestFactory()
+        post_data = {
+            'urgency': 'false'
+        }
+        request = request_factory.post('', post_data)
+        response = views.urgency(request, 12)
+        self.assertEqual(response.status_code, 302)
+
+    def test_urgency_description_get_request(self):
+        request_factory = RequestFactory()
+        request = request_factory.get('')
+        response = views.urgency_description(request, 12)
+        self.assertEqual(response.status_code, 200)
+
+    def test_urgency_description_post_request(self):
+        request_model = Request()
+        request_model.save()
+        request_factory = RequestFactory()
+        post_data = {
+            'urgency': 'this is very urgent'
+        }
+        request = request_factory.post('', post_data)
+        response = views.urgency_description(request, request_model.pk)
+        request_model.refresh_from_db()
+        self.assertEqual(request_model.urgency, 'this is very urgent')
+        self.assertEqual(response.status_code, 302)
+
+    def test_description_get_request(self):
+        request_factory = RequestFactory()
+        request = request_factory.get('')
+        response = views.description(request, 12)
+        self.assertEqual(response.status_code, 200)
+
+    def test_description_post_request(self):
+        request_model = Request()
+        request_model.save()
+        request_factory = RequestFactory()
+        post_data = {
+            'description': 'I want to buy some widgets'
+        }
+        request = request_factory.post('', post_data)
+        response = views.description(request, request_model.pk)
+        request_model.refresh_from_db()
+        self.assertEqual(request_model.description, 'I want to buy some widgets')
+        self.assertEqual(response.status_code, 302)
+
+    def test_submit_request_get_request(self):
+        request_factory = RequestFactory()
+        request = request_factory.get('')
+        response = views.submit_request(request, 12)
+        self.assertEqual(response.status_code, 200)
+
+    def test_submit_request_post_request(self):
+        request_model = Request()
+        request_model.save()
+        request_factory = RequestFactory()
+        post_data = {
+            'submit': 'true'
+        }
+        request = request_factory.post('', post_data)
+        response = views.submit_request(request, request_model.pk)
+        request_model.refresh_from_db()
+        self.assertEqual(request_model.submitted_at.__class__, datetime)
```

Comment:
I was thinking that this would compare the datetime from `post_data` to what's saved into the database, which is what the other tests do.